ui/apps/platform/src/types/imageIntegration.proto.ts

```diff
@@ -173,8 +173,10 @@ export type RhelImageIntegration = {
     type: 'rhel';
 } & BaseImageIntegration;
 
+export const scannerV4ImageIntegrationType = 'scannerv4';
+
 export type ScannerV4ImageIntegration = {
-    type: 'scannerv4';
+    type: typeof scannerV4ImageIntegrationType;
```

Comment:
I opted to make this an exportable constant instead of duplicating the string in the banner on the off chance that these values change and get out of sync. The property in the banner being compared is `string`, so TypeScript would not warn us of a change.